Cargo.toml

```diff
@@ -14,13 +14,6 @@ homepage = "http://maidsafe.net"
 [dependencies]
 rustc-serialize = "*"
 cbor = "*"
-# sodiumoxide = "0.0.4"
+sodiumoxide = "0.0.4"
```

Comment:
I think it will be from 0.0.6, I tried 0.0.5 from crates and got the same error.